### ISSUE TITLE
why Batch Normalization is under Conv part?

### ISSUE THREAD

**Issue by fyears** [zackchase/mxnet-the-straight-dope#151]
BN is kind of useful for mlp or cnn or even other networks. (Although, it's especially useful in very deep networks.)

Maybe it's better to put BN inside Part 3 DNN, analogous to dropout?

PS: If no one is writing this introduction, I am interested in contribute to this one. :-)

**Comment by zackchase** [comment]
I agree with both of these points. Go for it @fyears. Looking forward to the PR.

**Comment by fyears** [comment]
Hi, please review [my draft](https://github.com/fyears/mxnet-the-straight-dope/blob/add-batchnorm/P03-C05-mlp-batch-norm-scratch.ipynb). Am I on the right track? The implementation should be fine, but maybe some further explanations should be added...

**Comment by fyears** [comment]
Hi @zackchase finally I move the chapters back to cnn part because spatial bn needs extra considerations...

Once you merge my prs, you may close this issue. :-)
